### PR TITLE
[DOCFIX] Add the Chinese version of Running-Alluxio-on-GCE.md

### PR DESCRIPTION
**Pull request by huangshengbin426**

### CODE REVIEW

**Comment by ifcharming** [comment]
ok to test

File: docs/cn/Running-Alluxio-on-GCE.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+{% include Running-Alluxio-on-GCE/config-ssh.md %}
+
+复制 `deploy/vagrant/conf/gce.yml.template` to `deploy/vagrant/conf/gce.yml` :
+
+{% include Running-Alluxio-on-GCE/copy-gce.md %}
+
+在 `deploy/vagrant/conf/gce.yml`配置文件中,设置你的 项目 id,服务账号, JSON密钥的位置和已经创建好的ssh用户名.
+
+目前GCE默认使用 Hadoop2 作为它的底层文件系统, 因为GCS目前并不支持Alluxio。
```
Comment (ifcharming) [inline docs/cn/Running-Alluxio-on-GCE.md]: nit: 因为目前Alluxio并不支持GCS作为底层文件系统。

**Comment by ifcharming** [comment]
Thank you for the translation!
Overall looks good to me, only left one minor comment.

**Comment by huangshengbin426** [comment]
@ifcharming Thanks for review. Updated

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**